tableauserverclient/server/request_factory.py

```diff
@@ -555,6 +555,23 @@ def empty_req(self, xml_request):
         pass
 
 
+class WebhookRequest(object):
+    @_tsrequest_wrapped
+    def create_req(self, xml_request, webhook_item):
+        webhook = ET.SubElement(xml_request, 'webhook')
+        webhook.attrib['name'] = webhook_item.name
+
+        source = ET.SubElement(webhook, 'webhook-source')
+        event = ET.SubElement(source, webhook_item._event)
+
+        destination = ET.SubElement(webhook, 'webhook-destination')
+        post = ET.SubElement(destination, 'webhook-destination-http')
+        post.attrib['method'] = 'POST'
```

Comment:
Is there ever a need to make this configurable? Or will it always be POST?

If the latter, this is fine, otherwise we might want a `method` attribute on the model and pass through